cmd/root_test.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRootCommand(t *testing.T) {
+	fs := afero.NewMemMapFs()
+	configYaml, err := ioutil.ReadFile("./testdata/config.yaml")
+	err = afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
+	config.InitConfig(fs)
+	expectedOut := `astro is a command line interface for working with the Astronomer Platform.
```

Comment:
This test seems like it's going to be overly fragile - almost any change to the command would need a matching change in the test file -- i.e. I don't think having this test here gives us much value.

Reply:
yes and no, only when you add new command or remove you must update this test.